pkg/asset/installconfig/powervs/client.go

```diff
@@ -268,6 +271,34 @@ func (c *Client) GetInstanceCRNByName(ctx context.Context, name string, publish
 	return "", fmt.Errorf("DNS zone %q not found", name)
 }
 
+// GetDNSCustomResolverIP gets the DNS Server IP of a custom resolver associated with the specified VPC subnet in the specified DNS zone.
+func (c *Client) GetDNSCustomResolverIP(ctx context.Context, dnsID string, vpcID string) (string, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
```

Comment:
You're creating a child context but discarding it and passing the parent context to the calls inside this function. I'm guessing that was not your intention? From what I see below, the calls you pass the context to create their own child contexts, so you don't really need this.
```suggestion

```

Reply:
Yep makes sense, bad copy paste

Reply:
Removed other occurrences of it as well. Hope that's okay.